tests/test_main.py

```diff
@@ -11,5 +11,24 @@ def test_client():
 
 def test_GET_index(test_client):
     response = test_client.get("/")
-    assert b"This is index.html" in response.data
+    assert b"index.html" in response.data
+    assert response.status_code == 200
+
+
+def test_GET_search(test_client):
+    response = test_client.get("/search?query=サーカスTC")
+    assert response.data.decode("utf-8").count("サーカスTC") > 1
```

Comment:
等号不等号の向きは左側が小です。
ちなみに、検索結果が一つしかない場合は考慮しないんですか？

```suggestion
    assert 1 <= response.data.decode("utf-8").count("サーカスTC")
```